code/modules/cargo/bounties/assistant.dm

```diff
@@ -179,21 +162,13 @@
 	required_count = 5
 	wanted_types = list(/obj/item/toy/figure = TRUE)
 
-/datum/bounty/item/assistant/dead_mice
```

Comment:
If I may say, this bounty **IS** repeatable in regular gameplay. To be able to repeat the mouse bounty, players are encouraged to breed mice through slices of cheese, not just collecting them from maintenance. The reward is as high as it is due to the amount of time required to breed a whole set of mice, however.

Reply:
You know what, sure, readded